src/attribution/onINP.ts

```diff
@@ -63,7 +63,7 @@ let pendingLoAFs: PerformanceLongAnimationFrameTiming[] = [];
 let pendingEntriesGroups: pendingEntriesGroup[] = [];
 
 // The `processingEnd` time of most recently-processed event, chronologically.
-let latestProcessingEnd: number;
+let latestProcessingEnd: number = 0;
```

Comment:
I blame tsc for allowing this to be `number` without an initial value :P

Reply:
Yeah, a bit surprising that it didn't catch this.